dask/dataframe/tests/test_dataframe.py

```diff
@@ -3615,6 +3615,38 @@ def test_dataframe_reductions_arithmetic(reduction):
     )
 
 
+@pytest.mark.filterwarnings("ignore:UserWarning")
```

Comment:
Why are we ignoring warnings here? I don't think we want to.

Reply:
Gone